campus/support.html

```diff
@@ -0,0 +1,36 @@
+---
+title: Campus Support
+layout: text-optimized-width
+---
+
+<h1 class="mini-bar">{{ page.title }}</h1>
```

Comment:
Why this `mini-bar` class? It looks odd and I don’t see it on other pages. Unless there’s good reason for it, I’d like to drop the class.